triton/store.py

```diff
@@ -52,7 +54,7 @@ def flush(self):
             except OSError:
                 pass
 
-            self.writer = io.open(self.file_path, mode="wb")
+            self.writer = io.open(unicode_to_ascii_str(self.file_path), mode="wb")
```

Comment:
This change was made to patch a difference between local testing (`make test`) and drone testing. In local test environment (created by python virtualenv), `io.open` takes either a unicode or ascii string. In drone test environment, `io.open` requires and ascii string. I don't know why this is different, but this will temporarily patch the problem so that tests run properly in either environment.